portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -101,3 +116,91 @@ private String getChoice2(HttpServletRequest request) {
+// import com.google.appengine.api.datastore.Entity;
+// import java.io.IOException;
+// import com.google.sps.classes.Task;
+// import java.util.ArrayList;
+// import java.util.List;
+// import java.time.LocalTime; 
+// import java.util.Date;
+// import com.google.gson.Gson;
+// import javax.servlet.annotation.WebServlet;
+// import javax.servlet.http.HttpServlet;
+// import javax.servlet.http.HttpServletRequest;
+// import javax.servlet.http.HttpServletResponse;
+
+// /** Servlet that returns some example content. TODO: modify this file to handle comments data */
+// @WebServlet("/data")
+// public class DataServlet extends HttpServlet {
+
+//   private ArrayList<String> statements;
+
+//   public void init(){
+//       statements = new ArrayList<>();
+//   }
+  
+//   @Override
+//   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException{
+//     String json = convertToJsonUsingGson(statements);
+
+//     response.setContentType("application/json;");
+//     response.getWriter().println(json);
+
+//     Query query = new Query("Task").addSort("timestamp", SortDirection.DESCENDING);
+//     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+//     PreparedQuery results = datastore.prepare(query);
+
+//     ArrayList<String> stringOfComments = new ArrayList<>();  
+//     for (Entity entity : results.asIterable()) {
+//          String title = (String) entity.getProperty("title");
+//        //  String time = (String) entity.getProperty("timestamp");
+//          stringOfComments.add(title);
+//     }
+//   }
+
+//   @Override
+//    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+//     String name = getChoice(request);
+//     String comment = getChoice2(request);
+
+// //     Date dateNow = new Date();
+// //     long timeNow = dateNow.getTime();
+// //     int timeNowToUse = (int) timeNow;
+// //     String time = "6:00";
+// //   String title = request.getParameter("title");
+//     long timestamp = System.currentTimeMillis();
+//     String title = (name + ": " + comment);
+//     Entity taskEntity = new Entity("Task");
+//     taskEntity.setProperty("title", title);
+//     taskEntity.setProperty("timestamp", timestamp);
+
+//     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+//     datastore.put(taskEntity);
+
+//     response.sendRedirect("/index.html");
+//   }
+
+
+//   private String convertToJsonUsingGson(ArrayList state) {
+//     Gson gson = new Gson();
+//     String json = gson.toJson(state);
+//     return json;
+//   }
+
+//   private String getChoice(HttpServletRequest request) {
+//     String answerString = request.getParameter("name");
+//     return answerString;
+//   }
+//   private String getChoice2(HttpServletRequest request) {
+//     String answerString = request.getParameter("comment");
+//     return answerString;
+//   }
+// }
+
```

Comment:
Why is everything commented here?

Should this be deleted?

Reply:
Yes I need to delete that